packages/typography/package.json

```diff
@@ -14,12 +14,12 @@
   },
   "main": "build/index.js",
   "module": "build/index.es.js",
-  "style": "build/index.css",
   "files": [
     "build/*"
   ],
   "dependencies": {
-    "classnames": "^2.2.5",
+    "@hig/theme-context": "^1.0.1",
+    "emotion": "^10.0.4",
```

Comment:
I think it might be better to keep this at `10.0.0` for now as I think the other themeable components are at `10.0.0` and we can wait for Greenkeeper to have us update them all when the time comes.